docs/service-catalog/06-01-service-binding-usage.md

```diff
@@ -3,15 +3,15 @@ title: ServiceBindingUsage
 type: Custom Resource
 ---
 
-The `servicebindingusages.servicecatalog.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to inject Secrets to an application. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+The `servicebindingusages.servicecatalog.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to inject Secrets to an Application. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
```suggestion
The `servicebindingusages.servicecatalog.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to inject Secrets to an Application. To get the up-to-date CRD and show the output in the YAML format, run this command:
```
Please see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).